Dockerfile

```diff
@@ -53,16 +53,21 @@ RUN echo "@cloudposse https://apk.cloudposse.com/3.9/vendor" >> /etc/apk/reposit
 # Use TLS for alpine default repos
 RUN sed -i 's|http://dl-cdn.alpinelinux.org|https://alpine.global.ssl.fastly.net|g' /etc/apk/repositories && \
```

Comment:
I'm not beholden to this. Do you think we're better off moving all those to `rootfs/etc/apk/repositories`?

Reply:
I think the `sed` approach is better in that it is somewhat self-documenting about why we are not using the default repositories, and it updates itself when we upgrade Alpine.